examples/3.routes/8.tolls-and-ferries/queries.js

```diff
@@ -0,0 +1,97 @@
+import qql from 'graphql-tag';
+/*
+ * In this example a route from Aalborg, Denmakr to Oslo, Norway is requested.
+ * Origin and destination are required fields. An EV also needs to be selected.
+ * Only the EV ID here is mandatory, all other fields are optional and when not specified will use the default values.
+ * The changing conditions are:
+ *   - 80% SoC at Aalborg, Denmark
+ *   - EV can charge at CHAdeMO changers
+ *   - should use climate (temperature and weather conditions)
```

Comment:
You are saying we have 80% SOC in Oslo but it is not given in the newRoute mutation. 

I would add it to the mutation as we always recommend to give the state of charge